lib/algo/DMDc.cpp

```diff
@@ -208,6 +209,7 @@ void DMDc::takeSample(double* u_in, double t, double* f_in, bool last_step)
     d_sampled_times.push_back(sampled_time);
 }
 
+
```

Comment:
There a few extra lines that could be removed.

Reply:
Deleted the extra space